Bottom: c697d6221742e43b4e841037acca4faa7128b24e
Top:    ebe1505308448164a8b560ea7d92e4bf640ef517
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 18:02:27 +0000

Refresh of change-scroll-wheel-bindings

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 9b91a07..f4688ac 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -15,7 +15,7 @@
 #include "gui-pinout-preview.h"
 #include "pcb-printf.h"
 
-#include "hid/step/step_writer.h"
+#include "hid/common/step_id.h"
 #include "hid/common/quad.h"
 #include "hid/common/vertex3d.h"
 #include "hid/common/contour3d.h"
